src/factory/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 edition = "2021"
 name = "token-factory"
-version = "0.1.6"
+version = "0.24.0" #Will be replaced with the latest tag while building from the git
```

Comment:
This comment makes it sound like the build tool will modify the toml file.
Continuing this conversation in chat

Reply:
Done